### DISCUSSION TITLE
Scene Viewer's Online Previewer Tool Says 0 Errors and 0 Warnings, but the Model Won't Work

### DISCUSSION THREAD

**Discussion by NadimSKanaan** [google/model-viewer#1812] [Q&A]
Hello,

I looked at the file requirements for the model [here](https://developers.google.com/ar/develop/java/scene-viewer#file_requirements_for_models) and I asked a designer to create a model according to these specifications. I then went on to validate the model on [Scene Viewer's Online Previewer Tool](https://developers.google.com/ar/develop/java/scene-viewer#using_the_previewer_tool_to_validate_your_3d_models) and the console showed 0 errors and 0 warnings. When I launch Scene Viewer in my web-based AR prototype, it doesn't work. I converted my .glb to .usdz and it works fine in Quick Look!
What do you think is going wrong?
LINK TO MODEL: https://cdn.glitch.com/2ff2a36b-6d14-4205-b7ab-8b050166c154%2Fskin04_maya_unlit.glb?v=1610530137612

**Comment by elalish** [comment]
@tpsiaki: another SceneViewer GLB parsing issue?
